test/runtests.jl

```diff
@@ -22,3 +22,4 @@ include("test_pointevaluation.jl")
 # include("test_notebooks.jl")
 include("test_apply_rhs.jl")
 include("test_examples.jl")
+@test isa([Core.eval(Ferrite,type) for type in names(Ferrite)], Vector)  # Test that all exported symbols are defined
```

Comment:
I think there is a standard way of testing this, but can't find it right away.

Reply:
https://github.com/JuliaLang/julia/issues/18780?

Reply:
No, I was thinking of https://github.com/JuliaTesting/Aqua.jl/blob/master/src/exports.jl perhaps, but that seems a bit overkill, perhaps
```suggestion
@test all(x -> isdefined(Ferrite, x), names(Ferrite))  # Test that all exported symbols are defined

```
is sufficient.